interface/src/avatar/MyAvatar.h

```diff
@@ -519,6 +532,8 @@ class MyAvatar : public Avatar {
 
     /**jsdoc
      * <p>Specifies different avatar leaning and recentering behaviors.</p>
+     * <p class="important">Deprecated: This enumeration is deprecated and will be removed. If you need it, please contact 
+     *     the developers</p>
```

Comment:
```suggestion
     * <p class="important">Deprecated: This type is deprecated and will be removed.</p>
```
There's no such thing as an "enumeration" in JavaScript. This JSDoc is listed in the "Types" section of the MyAvatar API docs page.